apm-dist/release-docs/licenses/LICENSE-nacos.txt

```diff
@@ -0,0 +1,201 @@
+  Apache License
```

Comment:
This looks like a standard Apache license, which could not include the text. If I am right, please remove this file.

Reply:
i just found the okhttp license is same as nacos.
and okhttp license appear in apm-dist/release-docs/licenses/
so i put nacos license into there

Reply:
This is a suggestion, we don't need to copy all LICENSE when it is standard Apache 2.0.